python/lsst/sims/utils/ObservationMetaData.py

```diff
@@ -153,7 +166,7 @@ def summary(self):
         mydict['pointingDec'] = self.pointingDec
         mydict['rotSkyPos'] = self.rotSkyPos
 
-        mydict['mjd'] = self.mjd
+        mydict['mjd'] = self.mjd.TAI
```

Comment:
This is from the 'summary' property .. I just wanted to note that you get failures here if you don't set the boundLength and mjd (i.e. if they're Nones). I don't know if this is actually important though, but it might be worthwhile to test if they're None and then do something more explanatory. 

>>> from lsst.sims.utils import ObservationMetaData
>>> obs  = ObservationMetaData()
>>> obs.summary
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/lynnej/lsstRepos/sims_utils/python/lsst/sims/utils/ObservationMetaData.py", line 164, in summary
    mydict['boundLength'] = self.boundLength
  File "/Users/lynnej/lsstRepos/sims_utils/python/lsst/sims/utils/ObservationMetaData.py", line 505, in boundLength
    return numpy.degrees(self._boundLength)
AttributeError: 'NoneType' object has no attribute 'degrees'

>>> obs = ObservationMetaData(boundType='circle', boundLength=1.5)
>>> obs.summary
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/lynnej/lsstRepos/sims_utils/python/lsst/sims/utils/ObservationMetaData.py", line 169, in summary
    mydict['mjd'] = self.mjd.TAI
AttributeError: 'NoneType' object has no attribute 'TAI'

>>> obs = ObservationMetaData(boundType='circle', boundLength=1.5, mjd=49353.0)
>>> obs.summary
{'mjd': 49353.0, 'pointingRA': None, 'rotSkyPos': None, 'boundLength': 1.5000000000000002, 'skyBrightness': None, 'site': <lsst.sims.utils.Site.Site object at 0x104e2dfd0>, 'phoSimMetaData': None, 'boundType': 'circle', 'bandpass': 'r', 'pointingDec': None}
>>>